pyt/vulnerabilities/vulnerability_helper.py

```diff
@@ -56,16 +57,13 @@ def __init__(
         self.sink = sink
         self.sink_trigger_word = sink_trigger_word
 
-        self.reassignment_nodes = reassignment_nodes
-        self._remove_sink_from_secondary_nodes()
+        # Remove the sink node and all nodes after the sink from the list of reassignments.
```

Comment:
❤️ 
This is amazing, [it used to be more confusing](https://github.com/python-security/pyt/issues/46) but I forgot about the sink and after. Thanks so much for making this.